public/install.php

```diff
@@ -66,6 +66,14 @@
 			<?php endif ?>
 		</tr>
 		<tr>
+			<th>Composer Installed</th>
+			<?php if (is_dir($vendor_path) AND is_file($vendor_path.'autoload.php')): ?>
+				<td class="pass"><?php echo $vendor_path ?></td>
```

Comment:
http://programmers.stackexchange.com/questions/151661/is-it-bad-practice-to-use-tag-in-php

On Tue, Feb 10, 2015 at 5:50 AM, Alex Lenton <notifications@github.com>
wrote:

> In public/install.php
> <https://github.com/kohana/kohana/pull/54#discussion_r24396630>:
>
> > @@ -66,6 +66,14 @@
> >  			<?php endif ?>
> >  		</tr>
> >  		<tr>
> > +			<th>Composer Installed</th>
> > +			<?php if (is_dir($vendor_path) AND is_file($vendor_path.'autoload.php')): ?>
> > +				<td class="pass"><?php echo $vendor_path ?></td>
>
> This could use the short echo tags: <?=$vendor_path?>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kohana/kohana/pull/54/files#r24396630>.
>



-- 
*Gustavo A. Novaro*
Celular: +54 9 11 15-5757-7730
Blog: http://gustavonovaro.com.ar
Seguime en Twitter: @gnovaro <https://twitter.com/#%21/gnovaro>